src/tests/compiler/statement/doStatementTests.ts

```diff
@@ -0,0 +1,26 @@
+import * as ts from "typescript";
+import {expect} from "chai";
+import {DoStatement} from "./../../../compiler";
+import {getInfoFromText} from "./../testHelpers";
+
+function getInfoFromTextWithDoStatement(text: string) {
```

Comment:
I opened #209 because this would be useful to have in a reusable function.  I'll work on that one.